pkg/test/resourcefixture/testdata/sensitivefield/sensitivevaluefromsecret/_generated_object_sensitivevaluefromsecret.golden.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSSLCertificate
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/observed-secret-versions: (removed)
+    cnrm.cloud.google.com/project-id: ${projectId}
+    cnrm.cloud.google.com/state-into-spec: absent
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: computesslcertificate-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  certificate:
+    valueFrom:
+      secretKeyRef:
+        key: certificate
+        name: secret-${uniqueId}
+  location: global
+  privateKey:
+    valueFrom:
+      secretKeyRef:
+        key: privateKey
```

Comment:
May I ask what is the field `Key` refers to?

Reply:
It's defined in the test [create.yaml](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/f6ffa102c61d86b92572951aae7f385e89994569/pkg/test/resourcefixture/testdata/sensitivefield/sensitivevaluefromsecret/create.yaml#L32) as well.